[DOCS] Removes the SQL block config guides. (#7886)
great-expectations/great_expectations: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_airflow.md

> ### Connections and Backends

Reviewer: ```suggestion
### Connections and backends
```
![image](https://github.com/great-expectations/great_expectations/assets/104205257/172c3965-7aec-470f-be1c-16e36beae490)

> The `GreatExpectationsOperator` can run a checkpoint on a dataset stored in any backend compatible with Great Expectations. All that’s needed to get the Operator to point at an external dataset is to set up an [Airflow Connection](https://www.astronomer.io/guides/connections) to the datasource, and add the connection to your Great Expectations project, e.g. [using the CLI to add a Postgres backend](/docs/0.15.50/guides/connecting_to_your_data/database/postgres). Then, if using a `DataContextConfig` or `CheckpointConfig`, ensure that the `"datasources"` field refers to your backend connection name.

Reviewer: ```suggestion
The `GreatExpectationsOperator` can run a checkpoint on a dataset stored in any backend compatible with Great Expectations. All that’s needed to get the Operator to point at an external dataset is to set up an [Airflow Connection](https://www.astronomer.io/guides/connections) to the datasource, and add the connection to your Great Expectations project. For example, [using the CLI to add a Postgres backend](/docs/0.15.50/guides/connecting_to_your_data/database/postgres). If you're using a `DataContextConfig` or `CheckpointConfig`, ensure that the `"datasources"` field references the backend connection name.
```
![image](https://github.com/great-expectations/great_expectations/assets/104205257/130a7c57-d18e-4666-8c70-fff19d072cb2)
